experiments/configs/eval_configs/stem_eval_config.yaml

```diff
@@ -0,0 +1,11 @@
+model: hf-causal
+model_args: "pretrained=/fsx/path/checkpoint" # update
+tasks: "pile_pubmed-abstracts,pile_pubmed-central,headqa_en,sciq,pubmedqa,is_smiles,complete_smiles,periodic_table,openbookqa,hendrycksTest-college_biology,hendrycksTest-college_chemistry,hendrycksTest-college_mathematics,hendrycksTest-college_physics,hendrycksTest-high_school_mathematics,hendrycksTest-high_school_biology,hendrycksTest-high_school_chemistry,hendrycksTest-high_school_physics"
```

Comment:
cool, what are `pile_pubmed-abstracts, pile_pubmed-central, headqa_en`?

Reply:
Yes I hadn't spotted these before but I thought they looked relevant 
* pile_pubmed-abstracts, pile_pubmed-central: [perplexity tasks](https://github.com/OpenBioML/lm-evaluation-harness/blob/master/lm_eval/tasks/pile.py) based on those pile sub-datasets 

* HEAD-QA is a multi-choice HEAlthcare Dataset. The questions come from exams to
access a specialized position in the Spanish healthcare system, and are challenging
even for highly specialized humans. I just chose the english subset of this task.

Reply:
nice, they both sound great!

Reply:
means we also probably don't need to add a lambada task for next token over science papers